api/api/src/main/scala/uk/ac/wellcome/platform/api/elasticsearch/ElasticsearchQuery.scala

```diff
@@ -292,3 +293,18 @@ final case class BoolBoostedQuery(q: String) extends ElasticsearchPartialQuery {
     must(TitleQuery(q).elasticQuery).boost(1000)
   )
 }
+
+final case class ImageSimilarityQuery(q: String, index: Index)
```

Comment:
I don't think we ever do inheritance with case classes, is there a particular reason we are describing this as final?

Reply:
No idea, I was copying the convention that already exists in this file. fwiw I would like to revisit how query types are managed as we come back into search improvements work